pkg/certificate/signing_requestor_test.go

```diff
@@ -409,6 +415,35 @@ func awaitSecret(client dynamic.ResourceInterface) *corev1.Secret {
 		&corev1.Secret{})
 }
 
+func generateTestCSRForSigningRequestor() []byte {
+	// Generate a test private key
+	privateKey, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		Expect(err).NotTo(HaveOccurred())
+	}
```

Comment:
No need to check for nil, `Expect` does that.

```suggestion
	Expect(err).NotTo(HaveOccurred())
```